### PR TITLE
User guide feature part updated

### PR DESCRIPTION
**Pull request by nightism**
Lots of command formats in UserGuide.md have been changed.
All members should have a look as soon as possible and give critical reviews.

### CODE REVIEW

**Comment by nightism** [comment]
@potatowagon Please have a look of the new command format and update the Quick Start part in UserGuide.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
...
+* Task name could be a single word or a phrase, white space are allowed.
+
+* The floating task will be shown according to their `PRIORITY`.
+
+* Two floating tasks with the same `PRIORITY` will be shown according to
+  the order of the time they created.
 
 * The `PRIORITY` attribute is an integer which ranges from `0` to `5`.
 
 * The default piority of the floating task is 0.
```
Comment (pyokagan) [inline docs/UserGuide.md:162]: Since you are touching this area:
`s/piority/priority/`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 
 Adds a floating task to TaskTracker.
 
-    add "FLOATING_TASK_NAME" [PRIORITY]
+    task "FLOATING_TASK_NAME" [PRIORITY]
```
Comment (pyokagan) [inline docs/UserGuide.md:148]: What is the reason for requiring that the command word be differentiated from other kinds of tasks?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
...
 
 * Task name should be in a pair of quotation marks.
 
-* The floating task will be stored and shown to user according to the order of
-  the time they created.
+* Task name could be a single word or a phrase, white space are allowed.
```
Comment (pyokagan) [inline docs/UserGuide.md:153]: How will `"` be handled? Let's say I want to create a task name called `Here are some "quotes"`? How would I enter such a command?

Some ideas:
- We could use backslash escapes e.g. `task "Here are some \"quotes\""`
- Or we could use multiple quotes to surround the task name e.g. `task ""Here are some "quotes"""`

But I'm wondering which would be easier to type. What do you think?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -191,33 +191,45 @@ Adds an event with specific starting and ending date or time to TaskTracker.
...
   The default year of `dd/mm` is the current year and the default month for
-  `dd` is current month.
+  `dd` is the current month.
 
-* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCD` which
-  each letter represents a number ranges from 0 to 9. `AB` represent the hour
-  in 24 hour system, while `CD` represent the minute. Letter `A` can be omitted
-  if it is `0`.
+* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCDam` or `ABCDpm`
+  where each letter represents a number. `AB` represent the hour
```
Comment (pyokagan) [inline docs/UserGuide.md]: `represents a number` -> `represents a digit`?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -191,33 +191,45 @@ Adds an event with specific starting and ending date or time to TaskTracker.
...
+  in 12 hour system, while `CD` represent the minute. Letter `A` can only be omitted
```
Comment (pyokagan) [inline docs/UserGuide.md]: Likewise, `Letter A` -> `Digit A`?

File: docs/UserGuide.md (hunk 1 above)
Comment (nightism) [reply docs/UserGuide.md:162]: we just fetch the data from the database according to the order they are stored, no more extra work will be done here.
or we could use TASK_UNIQUE_INDEX to sort the task with the same priority

File: docs/UserGuide.md (hunk 2 above)
Comment (nightism) [reply docs/UserGuide.md:148]: since one single add may overload too much attribute, and problems such as the usability to distinguish `add "event" STARTING_DATE ENDING_TIME` and `add "due" DATE TIME` may be quite complex to be solved in the scenario.
Or we could add prefix like `st-` `et-` which are quite annoying

File: docs/UserGuide.md (hunk 3 above)
Comment (nightism) [reply docs/UserGuide.md:153]: At the very beginning, I think we could just inform the user not to use quotes in the name of the task and multiple quotes will be forbidden in the command
And we could inform the user to use ‘’ instead at this time

File: docs/UserGuide.md (hunk 4 above)
Comment (nightism) [reply docs/UserGuide.md]: noted, will be refined later

File: docs/UserGuide.md (hunk 5 above)
Comment (nightism) [reply docs/UserGuide.md]: noted

**Comment by nightism** [comment]
Some revision updated

File: docs/UserGuide.md (hunk 5 above)
Comment (pyokagan) [reply docs/UserGuide.md]: By the way, will we be implementing natural language dates e.g. `tomorrow`, `2 hours` etc.?

File: docs/UserGuide.md [hunk 6]
```diff
@@ -263,6 +263,25 @@ To create a deadline named `event proposal` with due time at today's 6 pm.
...
+
+List certain type of task stored in the database
+
+    list task|event|due
+    
+* `list task` will show all floating tasks according to the order of their priority.
+
+* `list event` will show all events according to the order of their starting time.
+
+* 'list due' will show all deadline according to their due time and due date. (when sorting
```
Comment (pyokagan) [inline docs/UserGuide.md]:
```
'list due' -> `list due`
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -263,6 +263,25 @@ To create a deadline named `event proposal` with due time at today's 6 pm.
...
+the deadline without due time, the sorting will use 11:59 pm of the due date as deafault)
```
Comment (pyokagan) [inline docs/UserGuide.md]: `deafault` -> `default`

File: docs/UserGuide.md [hunk 8]
```diff
@@ -263,6 +263,25 @@ To create a deadline named `event proposal` with due time at today's 6 pm.
...
+
+* Every tasks (floating tasks / events / deadlines) will have a unique index to differentiate
+their uniqueness. And their index will also be listed besides them when `list` command is
+excuted.
```
Comment (pyokagan) [inline docs/UserGuide.md]: `excuted` -> `executed`

File: docs/UserGuide.md [hunk 9]
```diff
@@ -263,6 +263,25 @@ To create a deadline named `event proposal` with due time at today's 6 pm.
 
 To create a deadline named `EE2024 homework 1` at 2016 1st October 6 am.
 
+### List tasks: `list`
+
+List certain type of task stored in the database
```
Comment (pyokagan) [inline docs/UserGuide.md:271]: Since our UI already has three panes for displaying events, deadlines and floating tasks, why not just have `list` list all of them at once?

File: docs/UserGuide.md [hunk 10]
```diff
@@ -286,19 +286,24 @@ excuted.
 
 Delete a useless floating task/event/deadline on TaskTracker.
 
-    del "TASK_NAME"
+    del TASK_UNIQUE_INDEX
 
-* Task name should be in a pair of quotation marks.
+* `TASK_UNIQUE_INDEX` will be different from each to each single task.
```
Comment (pyokagan) [inline docs/UserGuide.md:294]: How is the task's unique index generated?

File: docs/UserGuide.md [hunk 11]
```diff
@@ -305,61 +305,81 @@ List all the events stored in the database with their unique index number.
...
+* Quotation marks are not necessary for `NEW_NAME`.
 
 * `PRIORITY` should only be the interger ranges from `0` to `5`.
 
-* Here we assume that no duplicate task name is used among all the task.
-
 #### Examples
 
-* `edit "washing clothes" p-0`
+* `edit 00124 p-0`
```
Comment (pyokagan) [inline docs/UserGuide.md:328]: `00124` is a lot of digits to type :scream: If I get one digit wrong, then I edit the wrong task and will have to undo and type the edit command again. Each extra digit I type is an extra chance to get it wrong.

Why not just keep the current behaviour where the index is based on the position of the task in the list in the user interface?

File: docs/UserGuide.md [hunk 12]
```diff
@@ -456,6 +459,8 @@ Search task that contains specific keywords.
 
 * The `KEY_WORDS` are CASE-SENSITIVE
 
+* Quotations marks are not allowed in `KEY_WORLDS`.
```
Comment (pyokagan) [inline docs/UserGuide.md]: I think you meant `KEY_WORDS`?

File: docs/UserGuide.md [hunk 13]
```diff
@@ -237,7 +239,8 @@ Adds a deadline with specific due date or time to TaskTracker.
 
     due “DEADLINE_NAME” <DATE> <TIME>
 
-* Dealine name should be in a pair of quotation marks.
+* Dealine name should be in a pair of quotation marks. And quotations marks are
```
Comment (pyokagan) [inline docs/UserGuide.md:242]: `Deadline`

File: docs/UserGuide.md [hunk 14]
```diff
@@ -183,152 +193,215 @@ Adds an event with specific starting and ending date or time to TaskTracker.
...
+* `edit 00125 n-"go to Nanyang Mart" p-1`
+
+Edit task with unique index of `00125`'s name to `go to Nanyang Mart` and priority to 1.
+
+#### Edit an event :
 
 Edit an event to revise its name, starting/ending date/time and location.
 
-    edit "TASK_NAME" [sd-NEW_START_TIME | st-NEW_START_DATE | ed-NEW_END_DATE | et-NEW_END_TIME | n-NEW_NAME | loc-NEW_LOCATION]...
+    edit TASK_UNIQUE_INDEX [sd-NEW_START_TIME | st-NEW_START_DATE | ed-NEW_END_DATE | et-NEW_END_TIME | n-NEW_NAME | loc-NEW_LOCATION]...
```
Comment (pyokagan) [inline docs/UserGuide.md:344]: Are users allowed to specify flags in any order?

I think they should be allowed to specify flags in any order, but that needs to be stated in the User Guide. (Unless that is not true?)

File: docs/UserGuide.md [hunk 15]
```diff
@@ -183,152 +193,215 @@ Adds an event with specific starting and ending date or time to TaskTracker.
...
   The default year of `dd/mm` is the current year and the default month for
-  `dd` is current month.
+  `dd` is the current month.
 
-* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCD` which
-  each letter represents a number ranges from 0 to 9. `AB` represent the hour
-  in 24 hour system, while `CD` represent the minute. Letter `A` can be omitted
-  if it is `0`.
+* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCDam` or `ABCDpm`
+  where each letter represents a digit. `AB` represent the hour
```
Comment (pyokagan) [inline docs/UserGuide.md:199]: How about rewording it as `AB represents the hour (1-12), while CD represents the minute (00-60)`?

File: docs/UserGuide.md [hunk 16]
```diff
@@ -183,152 +193,215 @@ Adds an event with specific starting and ending date or time to TaskTracker.
...
+  if it is `0`, and 'CD' can be only omitted (together) when both of them are `0`.
 
-* The default starting date will be the same as the real date on user's
-  computer.
+* The default starting date will be the current date.
 
 * The default ending date will be the same as the starting date.
 
-* The default starting time will be `0000`.
+* The default starting time will be `0000am` of starting date.
```
Comment (pyokagan) [inline docs/UserGuide.md:207]: `12am`? Is `0000am` even accepted with the current command format?

File: docs/UserGuide.md [hunk 17]
```diff
@@ -183,152 +193,215 @@ Adds an event with specific starting and ending date or time to TaskTracker.
...
+* `event "programming workshop" 10am 5pm loc-LT15`
+
+To create an event `programming workshop` with starting time at 10am of current date
+and end at 5pm. The location will be LT15.
 
-* `add "programming workshop" 1000 1700 loc-LT15`
+* `event "sports training camp" 1/12/2016 10/12/2016`
 
-* `add "sports training camp" 1/12/2016 10/12/2016`
+To create an event `sports training camp` with starting time at 2016 1st December 0am
```
Comment (pyokagan) [inline docs/UserGuide.md:227]: `1st December 12am`? I don't think `0am` should even be accepted.

File: docs/UserGuide.md [hunk 18]
```diff
@@ -183,152 +193,215 @@ Adds an event with specific starting and ending date or time to TaskTracker.
...
 
-* `del "CS2102 week 8 lecture"`
+Delete the task with the unique index of `00123`.
 
-* `del "reading a book"`
+### Edit a floating task/event/dealine: `edit`
 
-### Edit a floating task: `edit`
+* Edit command can only edit the parameters of each commands but cannot transform a task to another
+type. For example, `eidt` cannot transform a floating task to a event task.
```
Comment (pyokagan) [inline docs/UserGuide.md:314]: `edit`

File: docs/UserGuide.md [hunk 19]
```diff
@@ -346,29 +419,58 @@ Show all empty time slots in a given time period with a given duration.
...
-View all the task of a specific date in the Calander View of TaskTracker.
+View all the deadlines and events of a specific date in TaskTracker.
 
     view DATE
+    
+* `DATE` will be the same format of what we use to create event. Please refer to `event`
+command for details.
+
+* The deadlines and events will be listed according to the order of deadlines' due time and
+evnets' starting time.
```
Comment (pyokagan) [inline docs/UserGuide.md:445]: `events'`

File: docs/UserGuide.md (hunk 17 above)
Comment (pyokagan) [reply docs/UserGuide.md:227]: Or maybe we could accept `0am`, but the user guide should most definitely use `12am` because that's what people normally use.

File: docs/UserGuide.md [hunk 20]
```diff
@@ -183,152 +193,215 @@ Adds an event with specific starting and ending date or time to TaskTracker.
...
+  `dd` is the current month.
 
-* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCD` which
-  each letter represents a number ranges from 0 to 9. `AB` represent the hour
-  in 24 hour system, while `CD` represent the minute. Letter `A` can be omitted
-  if it is `0`.
+* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCDam` or `ABCDpm`
+  where each letter represents a digit. `AB` represent the hour
+  in 12 hour system, while `CD` represent the minute. Digit `A` can only be omitted
+  if it is `0`, and 'CD' can be only omitted (together) when both of them are `0`.
```
Comment (pyokagan) [inline docs/UserGuide.md:201]: Maybe allow users to have an optional period or colon for times as well? e.g. `2.30am` or `2:30am`. I'm much more used to typing timings with a period and it looks more natural to me,

**Review by pyokagan — APPROVED** [review]
This PR is a definite improvement and so I've marked it for merging tomorrow.

However, there are still some points which need to be clarified which I've pointed out in the review. These points need to be addressed in a future PR. Particularly, I don't really like the use of unique task IDs as it could lead to lots of typing of digits when there are a lot of tasks in the database.

Thanks for breaking the changes up into bite-sized commits, by the way. It makes for a pleasant read.

**Comment by pyokagan** [comment]
I'll also probably add a commit on top to fix all the spelling errors ><

**Comment by pyokagan** [comment]
@nightism Eh I just noticed you marked this as priority high so I guess I've to go merge it now

**Comment by pyokagan** [comment]
@nightism Ugh, I just noticed that for all your commit messages, your commit subject is not separated from the commit body. Make sure you separate it with two newlines, like this:

```
Commit Subject

Commit Message Body
```

File: docs/UserGuide.md (hunk 11 above)
Comment (nightism) [reply docs/UserGuide.md:328]: since during the tutorial, the tutor suggests a scenario where the task name is extremely long, and if there is one digit wrong the user has to retype which is even worse than doing an undo

File: docs/UserGuide.md (hunk 5 above)
Comment (nightism) [reply docs/UserGuide.md]: not at this time I think, we may consider to add them when we get our minimum viable product

File: docs/UserGuide.md (hunk 6 above)
Comment (nightism) [reply docs/UserGuide.md]: solved

File: docs/UserGuide.md (hunk 7 above)
Comment (nightism) [reply docs/UserGuide.md]: solved

File: docs/UserGuide.md (hunk 8 above)
Comment (nightism) [reply docs/UserGuide.md]: solved

File: docs/UserGuide.md (hunk 9 above)
Comment (nightism) [reply docs/UserGuide.md:271]: I think this command should only be finalised when we finalise our UI design, otherwise, we may not be able to decide on how this command really works.
Since for our first design, deadlines and floating tasks are already listed on the UI

File: docs/UserGuide.md (hunk 10 above)
Comment (nightism) [reply docs/UserGuide.md:294]: Basically, it will start from 0 or (00000), and  with each one task added, the number will increase by 1. It is just a counter in other words. But I think it is not necessary to explain this in UserGuide, and that why I did not explain here.

File: docs/UserGuide.md (hunk 12 above)
Comment (nightism) [reply docs/UserGuide.md]: solved

**Comment by pyokagan** [comment]
Merged in 77e9d7927ad6c1812bfaa36dee9c5516e3cdc7e1

**Comment by pyokagan** [comment]
@nightism Do the new changes in a separate PR.

File: docs/UserGuide.md (hunk 1 above)
Comment (pyokagan) [reply docs/UserGuide.md:162]: As I said below, I do not like the idea of `TASK_UNIQUE_INDEX`. Sorting by insertion order in the database is a decent idea though.

File: docs/UserGuide.md (hunk 3 above)
Comment (pyokagan) [reply docs/UserGuide.md:153]: ~~On one hand, we could argue that users would not usually use double-quotes anyway. On the other hand, I think that there needs to be a way to escape the quotes just to provide the capability on the off-chance that users really need to use double-quotes.~~
Edit: Ah nevermind, I think it's okay to leave it like this for now.

File: docs/UserGuide.md (hunk 9 above)
Comment (pyokagan) [reply docs/UserGuide.md:271]: > Since for our first design, deadlines and floating tasks are already listed on the UI

Yeah, that's what I said. Since our first design, we already have events, deadlines and floating task boxes in our UI. So why not the `list` command use these boxes to list events, deadlines and floating tasks all at once?

File: docs/UserGuide.md (hunk 10 above)
Comment (pyokagan) [reply docs/UserGuide.md:294]: This kind of design will need to come with a system that reuses the IDs of previous tasks.

If not, after using the app for a long time, we are going to end up with task IDs of `342823` and more.

File: docs/UserGuide.md (hunk 11 above)
Comment (pyokagan) [reply docs/UserGuide.md:328]: Yes, I agree that this is an improvement over having the user type the name of the tasks.

However, I suggested an even better idea:

> Why not just keep the current behaviour where the index is based on the position of the task in the list in the user interface?

Currently, with our UI design, we only see a small number of tasks at any one time. E.g., we see the events that happen today (which is a small number), we see the deadlines that happen soon (should be a small number). So, with the above, the user will likely only need to type at most 2 digits.

Contrast it with your "unique ID" design, where the user will likely end up having to type 3 digits or more once the user starts storing over a hundred tasks. I'm sure we can agree that my solution will lead to the user having to type _less_ digits on average.

File: docs/UserGuide.md (hunk 16 above)
Comment (pyokagan) [reply docs/UserGuide.md:207]: I think we could accept `0000am`, but the user guide should use `12am` to not confuse the reader.

Comment (potatowagon) [reply docs/UserGuide.md:207]: 0000am is not user friendly and intuitive

File: docs/UserGuide.md (hunk 2 above)
Comment (potatowagon) [reply docs/UserGuide.md:148]: An alternative sololution is we force the user to include date and time, but we provide shortcuts , which would still be conveinient after all. This means we dont assume and exclude any of the fields: start time, start date, end time, end date,  must all be included in the command format. for example,
floating task has to have start tie and start date, but can be written as 3pm tdy instead fo 3pm 10oct2016
events can be written as 3pm tdy 6pm tmr, it is still convienient

**Comment by potatowagon** [comment]
@nightism i will merge your work and do the updates. Your feature updates will be reflected in the next pull request. The pull request from yesterday doesnt have your changes yet, still familiarizing myself with this issues system

File: docs/UserGuide.md (hunk 1 above)
Comment (nightism) [reply docs/UserGuide.md:162]: but it will be inconvenient for the user to delete or edit a task if we do not use an index to address it

File: docs/UserGuide.md (hunk 2 above)
Comment (nightism) [reply docs/UserGuide.md:148]: strings like "10oct2016" or "3pm tdy 6pm" needs lots of work to be done in the parser.
since I am still working on the fundamentals functionality of the parser, currently I do not have any idea on how we could parse strings mentioned above

File: docs/UserGuide.md (hunk 9 above)
Comment (nightism) [reply docs/UserGuide.md:271]: > So why not the list command use these boxes to list events, deadlines and floating tasks all at once? <br>

But for our first design, we do not even have to use `list`, since all the tasks will be shown automatically on the ui once they are created. And if the user wants to see tasks for a certain day, they could use `view DATE` command.

File: docs/UserGuide.md (hunk 10 above)
Comment (nightism) [reply docs/UserGuide.md:294]: > This kind of design will need to come with a system that reuses the IDs of previous tasks.

Yes, I agree. We could let the index starts from the beginning again once the index reaches the upper bond.

File: docs/UserGuide.md (hunk 11 above)
Comment (nightism) [reply docs/UserGuide.md:328]: Yes, that could be an idea. But we also has to settle the ui design first.

File: docs/UserGuide.md (hunk 14 above)
Comment (nightism) [reply docs/UserGuide.md:344]: Neither "allow" or "not allow" can be achieved in the parser. But if you think the order should be fixed, then I will modify this part later.

File: docs/UserGuide.md (hunk 15 above)
Comment (nightism) [reply docs/UserGuide.md:199]: noted, will be revised

File: docs/UserGuide.md (hunk 16 above)
Comment (nightism) [reply docs/UserGuide.md:207]: 1. `0000am` could be written as `0am` as indicated in one of the previous descriptions
2. `12am` is the start of a day or the end? For me, I think 12am should be the start, but I think it will be more reasonable to set the start as 0.

File: docs/UserGuide.md (hunk 17 above)
Comment (nightism) [reply docs/UserGuide.md:227]: Emm, okay, we could use both notations.

File: docs/UserGuide.md (hunk 18 above)
Comment (nightism) [reply docs/UserGuide.md:314]: noted

File: docs/UserGuide.md (hunk 20 above)
Comment (nightism) [reply docs/UserGuide.md:201]: So if we allow the colon, will notations like `230am` still be accepted? Or we just force the user to use colon for all time notations?

File: docs/UserGuide.md (hunk 10 above)
Comment (pyokagan) [reply docs/UserGuide.md:294]: So what's the upper bound?

File: docs/UserGuide.md [hunk 21]
```diff
@@ -141,34 +141,44 @@
...
-* `add "ProgressReflection"`
+To create a task called `ProgressReflection` with default `PRIORITY` of 0.
 
-### Adding a event-like task : `add`
+### Adding a event-like task : `event`
 
 Adds an event with specific starting and ending date or time to TaskTracker.
 
-    add “EVENT_NAME” <STARTING_DATE> <STARTING_TIME> <ENDING_DATE> <ENDING_TIME> [loc/LOCATION]
+    event “EVENT_NAME” <STARTING_DATE> <STARTING_TIME> <ENDING_DATE> <ENDING_TIME> [loc-LOCATION]
```
Comment (pyokagan) [inline docs/UserGuide.md:178]: @nightism Okay, I've implemented the `event` command in #37 (with the exception of `location`) and it seems to work well, so I think this command design is good enough,

File: docs/UserGuide.md (hunk 9 above)
Comment (pyokagan) [reply docs/UserGuide.md:271]: > But for our first design, we do not even have to use list, since all the tasks will be shown automatically on the ui once they are created.

That's right, so we probably don't even need a `list` command.

> And if the user wants to see tasks for a certain day, they could use view DATE command.

I don't see how this statement is relevant to the issue at hand.

File: docs/UserGuide.md (hunk 14 above)
Comment (pyokagan) [reply docs/UserGuide.md:344]: > But if you think the order should be fixed, then I will modify this part later.

????

I said:

> I think they (the users) should be allowed to specify flags in any order

File: docs/UserGuide.md (hunk 16 above)
Comment (pyokagan) [reply docs/UserGuide.md:207]: > 12am is the start of a day or the end? For me, I think 12am should be the start, but I think it will be more reasonable to set the start as 0.

It's not really what is "technically correct". If we want to be technically correct, we should not be using 12am or 0am at all since they are not valid 12-hour clock timings.

Rather, it is about convention, and that means looking at what most apps already do. I think, for example, that the Windows clock uses 12am to refer to midnight (the start of the day).

File: docs/UserGuide.md (hunk 17 above)
Comment (pyokagan) [reply docs/UserGuide.md:227]: Yeah, I just checked, and the Java DateTime parser already accepts both `0am` and `12am`.

File: docs/UserGuide.md (hunk 20 above)
Comment (pyokagan) [reply docs/UserGuide.md:201]: For simplicity, we can just leave it out for now. I'll bring it up again in a few days time.

File: docs/UserGuide.md (hunk 2 above)
Comment (pyokagan) [reply docs/UserGuide.md:148]: Yeah, we could leave it like this for now.

I just implemented the add event command in #37, and I think the syntax is decent enough.

Comment (pyokagan) [reply docs/UserGuide.md:148]: Okay, with one technical problem: the java.time.[DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html) that I'm using does not support times like `400pm`. It needs to be like `4:00pm` instead. I'll send a PR to fix this in the User Guide.

File: docs/UserGuide.md (hunk 1 above)
Comment (pyokagan) [reply docs/UserGuide.md:162]: We'll continue the discussion in https://github.com/CS2103AUG2016-T11-C4/main/pull/39#discussion_r82523900

File: docs/UserGuide.md (hunk 20 above)
Comment (pyokagan) [reply docs/UserGuide.md:201]: Oh, whoops. Turns out the Java's [DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html) does not support notations like `230am`. We have to use `2:30am` instead.

Comment (pyokagan) [reply docs/UserGuide.md:201]: For now, until we implement a better parser.

File: docs/UserGuide.md (hunk 10 above)
Comment (Bogi1001) [reply docs/UserGuide.md:294]: For the task's unique index, we could instead use a format like "Day.Month.Year.Number". For example, if I'm entering a 4th task on 3rd May 2016, the task's unique index will be generated as: "3M164".....or "3.M.16.4"...
This way the upper bound needs to be set only for a specific day....which is much easier than setting an upper bound for the whole period when the user is using the app
Also, it can be easier to sort tasks based on days

**Comment by Bogi1001** [comment]
In point 10, to delete a task/ event....the command is : del "cupcake festival".....what if the user wants to delete an event on another date? Can we include an easier alternative for "del TASK_UNIQUE_INDEX"....?
